.github/workflows/angular.yml

```diff
@@ -73,12 +73,12 @@ jobs:
                       protractor: 1
                     - app-type: ngx-mongodb-kafka-cucumber
                       entity: mongodb
-                      profile: dev
+                      profile: dev,webpack
```

Comment:
I don't understand why it is needed here ?

Reply:
It's due to the fact that we moved the webpack profile as default activation: if the user specifies another profile (here 'dev')while triggering the build, default activation is not taken into account anymore.
Here, we could have specified nothing in the profile section which would have been equivalent to activate 'dev, webpack', but it wouldn't have worked for gradle builds

Reply:
so it means `mvn package -Pdev` doesn't work anymore ?
In this case, we need to update https://github.com/hipster-labs/jhipster-daily-builds too

Reply:
Let's update it!

Reply:
I plan to do the release tomorrow afternoon, if you can update the daily builds before merging this, it would be nice
Otherwise this PR will be included in another release

Reply:
This one? https://github.com/hipster-labs/jhipster-daily-builds/pull/38

Reply:
thanks, I put some minor comments there